presentation/src/main/java/com/going/presentation/todo/editinfo/BottomSheetEditDateFragment.kt

```diff
@@ -0,0 +1,87 @@
+package com.going.presentation.todo.editinfo
+
+import android.os.Bundle
+import android.view.View
+import com.going.presentation.R
+import com.going.presentation.databinding.FragmentBottomSheetEditDateTripBinding
+import com.going.ui.base.BaseBottomSheet
+import com.going.ui.extension.setOnSingleClickListener
+import java.util.Calendar
+
+class BottomSheetEditDateFragment(val viewModel: EditTripInfoViewModel, val isStart: Boolean) :
+    BaseBottomSheet<FragmentBottomSheetEditDateTripBinding>(R.layout.fragment_bottom_sheet_edit_date_trip) {
+
+    override fun onStart() {
+        super.onStart()
+        dialog?.window?.setBackgroundDrawableResource(R.color.transparent)
+        if (isStart)
+            customStartDate()
+        else customEndDate()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        binding.lifecycleOwner = viewLifecycleOwner
+        initFinishBtnClickListener()
+    }
+
+    private fun customStartDate() {
+        val datePicker = binding.dpEditTripDate
+        val calendar = Calendar.getInstance()
+
+        val currentStartYear = calendar.get(Calendar.YEAR)
+        val currentStartMonth = calendar.get(Calendar.MONTH)
+        val currentStartDay = calendar.get(Calendar.DAY_OF_MONTH)
```

Comment:
요 친구들은 맨 아래에 `datePicker.updateDate(currentStartYear, currentStartMonth, currentStartDay)` 에서만 사용되는 것 같은데, 불필요한 변수 선언 대신 스코프 함수로 처리해보는건 어떨까요! (get때문에 안돌아갈수도 있긴 함)
```suggestion
val calendar = Calendar.getInstance()
val datePicker = binding.dpEditTripDate.apply {
    updateDate(
        calendar.get(Calendar.YEAR),
        calendar.get(Calendar.MONTH),
        calendar.get(Calendar.DAY_OF_MONTH)
    )
}
```
이런 느낌으로요!

Reply:
좋은 지적 감사합니다!!! 이렇게 스코프로 수정했습니다!